src/main/resources/simulator/vpi.h

```diff
@@ -32,7 +32,7 @@ class vpi_api_t: public sim_api_t<vpiHandle> {
     vpiHandle syscall_handle = vpi_handle(vpiSysTfCall, NULL);
     vpiHandle arg_iter = vpi_iterate(vpiArgument, syscall_handle);
     // Cache Inputs
-    if(arg_iter != nullptr) {
+    if(arg_iter != NULL) {
```

Comment:
this seems reasonable.

Probably this is somehow compiled by an old compiler which does not support C++11 (https://en.cppreference.com/w/cpp/language/nullptr) by default.